src/jobsautoreport/slack.py

```diff
@@ -42,62 +45,83 @@ def _upload_file(
         filename: str,
         thread_time_stamp: Optional[str],
     ) -> None:
-        response = self._client.files_upload(
-            channels=[self._channel_id],
-            file=file_path,
-            filename=filename,
-            initial_comment=file_title,
-            thread_ts=thread_time_stamp,
-        )
-        response.validate()
-        logger.info("%s was uploaded successfully", filename)
+        TRIES = 3
```

Comment:
Cant we use [retry package](https://pypi.org/project/retry/) instead?

Reply:
+1

also, do we really need this kind of retry? I don't remember seeing any problem uploading files to slack in other places we do that

Reply:
Lately we have been getting timeouts on upload tries to slack in stage environment from time time to time, see https://redhat-internal.slack.com/archives/C02L26KMHMW/p1681999420141539. I think if we can handle it, it would be better then letting the pod restart. WDYT?

Reply:
yeah it's probably better to retry in that case
let's have it in another PR either way